demo-project/demo-project-integration-test/README.md

```diff
@@ -0,0 +1,35 @@
+# Demo spring boot application - Integration testing
+
+This folder demonstrates how to do integration testing of the Demo application using Docker and Docker Compose.
+
+The [docker-compose.yml](./docker-compose.yml]) file defines 3 Docker containers: Spring Boot, Postgresql, and RabbitMq. 
+
+# How to run
+## 1. Create docker image
+First you have to create a Docker image for the Demo Spring Boot application.
+This is done by issuing a Maven command:
+```sh
+cd demo-project #make sure to cd to the .. folder
+mvn spring-boot:build-image -Dspring-boot.build-image.imageName=logb/demo-project
+```
+This will build a Docker image with the name logb/demo-project:latest.
```

Comment:
We should investigate versioning a little. Like, doesn't the maven version get applied as the docker tag as well? In general, working with `latest` on anything else than locally has a high risk of grabbing the wrong version.

Reply:
This works: mvn spring-boot:build-image -Dspring-boot.build-image.imageName=logb/demo-project:**${project.version}**. However we still need to update the reference to the image in the docker-compose.yml file. Is this acceptable?